winui/Kanban/overview.md

```diff
@@ -11,12 +11,17 @@ documentation: ug
 
 The Syncfusion WinUI Kanban control provides an efficient way to visualize workflows at each stage of completion. It enables clear planning and offers a transparent view of work progress. With a range of essential features, SfKanban is ideal for monitoring tasks throughout the software development cycle.
 
+![Overview in WinUI Kanban control](overview-images/overview-in-winui-kanban-control.png)
```

Comment:
Add images folder, inside create overview folder and then add images

images->overview->overview-in-winui-kanban-control.png